recipes/pyflame/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "pyflame" %}
+{% set version = "1.5.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev: v{{ version }}
+  git_url: https://github.com/uber/pyflame
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - autoconf
+    - python
+
+  run:
+    - python
+
+test:
```

Comment:
You can use the `commands` element instead of the `run_test.sh`:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands